.gitignore

```diff
@@ -47,3 +47,5 @@ build/Release
 node_modules
 yarn.lock
 
+.vscode
+output
```

Comment:
Kept noticing this random `output` file, must have sneaked in somehow this can prevent that.